pkg/util/winutil/etw/etw.go

```diff
@@ -120,10 +154,10 @@ func StartEtw(subscriptionName string, etwProviders ProviderType, sub Subscriber
 //
 // See above note about http-centrism
 func StopEtw(subscriptionName string) {
-	if len(subscribers) != 0 {
+	if safeSubscribers.Len() != 0 {
```

Comment:
Similar comment here, the number of items was zero while the lock was held, but it might have changed concurrently once the lock is released